src/library_gl.js

```diff
@@ -4690,3 +5001,5 @@ recordGLProcAddressGet(LibraryGL);
 
 // Final merge
 mergeInto(LibraryManager.library, LibraryGL);
+
+assert([ USE_WEBGL2, USE_WEBGL2_COMPUTE ].filter(function(element) { return element; }).length <= 1, 'cannot resolve conflicting USE_WEBGL* settings');
```

Comment:
It seems fine to enable both of these options. USE_WEBGL2_COMPUTE would win.

However it doesn't really matter to me, you can keep this if you want.